packages/migration/src/migrations/v0030_mark3pVideoAsMuted.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import type { ResourceV0, ResourceType } from '../types';
+import type {
+  GifElementV29,
+  ImageElementV29,
+  PageV29,
+  ProductElementV29,
+  ShapeElementV29,
+  StoryV29,
+  TextElementV29,
+  UnionElementV29,
+  VideoElementV29,
+} from './v0029_unifyGifResources';
+
+export type TextElementV30 = TextElementV29;
+export type ProductElementV30 = ProductElementV29;
+export type ShapeElementV30 = ShapeElementV29;
+export type ImageElementV30 = ImageElementV29;
+
+// @todo Should we drag resource versions along in each file? Currently, basing on ResourceV0 and extending fully per file.
```

Comment:
Waiting for thoughts about this... Currently creating a new Resource for each element is done starting from V31, should we do it since V1 for each file even though it's not used much there?

Reply:
Seems like a lot of repetition. If we can avoid that, why not :)